config/services.yml

```diff
@@ -249,6 +249,36 @@ https://cmr.earthdata.nasa.gov:
       - image: !Env ${CMR_GRANULE_LOCATOR_IMAGE}
       - image: !Env ${PODAAC_L2_SUBSETTER_IMAGE}
 
+  - name: sds/HOSS
+    # HOSS provides variable and bounding box spatial subsetting (geographically gridded collections)
+    data_operation_version: '0.11.0'
+    type:
+      <<: *default-argo-config
+      params:
+        <<: *default-argo-params
+        template: var-subsetter
+        template_type: chaining
+        image: !Env ${VAR_SUBSETTER_IMAGE}
+        image_pull_policy: !Env ${VAR_SUBSETTER_IMAGE_PULL_POLICY}
+        parallelism: !Env ${VAR_SUBSETTER_PARALLELISM}
+        env:
+          <<: *default-argo-env
+          STAGING_PATH: public/sds/HOSS
+    # umm_s:
```

Comment:
Has the service record not been created yet?

I'm sure you realize it, but as is we won't be able to use HOSS for any collections in production. I'm wondering if there might be a test collection in production that we'd want to make available - something like we do with harmony-service-example to verify the service is working correctly in production.

Reply:
I have the same question.

Reply:
Good point, Chris.

To my knowledge a HOSS UMM-S entry does not yet exist in the GES-DISC production CMR provider. There are two objectives associated with getting HOSS in production and supporting the GPM/IMERG collection, one for Data Services and the other for GES-DISC; I think the UMM-S record should fall under the GES-DISC objective. The Data Services team don't have the requisite permissions to make this UMM-S record in the GES-DISC provider, and don't have an EEDTEST provider in production to make a UMM-S record of our own.

I was hoping by popping this comment in the `services.yml` it would make it super easy for a GES-DISC developer (maybe @nasajay in a couple of weeks from now) to pop the UMM-S concept ID here when the record exists.

I think it would be _really_ nice to have some sort of test collection, though! I'd be interested in talking to you guys about how you got around the lack of an EEDTEST provider for that collection for harmony-service-example.

Reply:
In production there's a provider with collections and granules that are hidden from normal users via ACLs - XYZ_PROV. 

We have a test collection there that we use with harmony-service-example: C1756916832-XYZ_PROV.

I'm fine with merging this in as is, just making sure you realized there wouldn't be any way to verify HOSS working with harmony in production without at least one collection.

Reply:
`XYZ_PROV` sounds promising! I'll check within our team about making some test data available from this provider. If that's something we want to pursue (and it sounds great to me) I'll probably then coordinate with you offline about getting the correct access to the provider so we can do that. Thanks, Chris!

In the meantime, though, my inclination is to forge on ahead with this PR, albeit with a production HOSS that is currently non-verifiable until we have  a UMM-S and associated collection.

Reply:
- [ ] Add `XYZ_PROV` UMM-S record.